Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -387,9 +387,76 @@ namespace xr
             renderResource.DepthSwapchains.clear();
         }
 
+        Anchor CreateAnchor(Pose pose, NativeTrackablePtr trackable, XrTime time)
+        {
+            auto& apiExtensions = HmdImpl.Extensions;
+
```

Comment:
nit: consider const auto& to avoid accidentally setting the extensions unique_ptr

Reply:
Could also dereference the pointer if you wanted to avoid arrow syntax.

```
const auto& apiExtensions = *HmdImpl.Extensions;
```